### PR TITLE
Reading history

### PR DESCRIPTION
**Pull request by khanhkhanh**

### CODE REVIEW

File: app/controllers/readings_controller.rb [hunk 1]
```diff
@@ -1,6 +1,12 @@
 class ReadingsController < ApplicationController
   before_action :authenticate_user!
 
+  def show
+    @user = User.find params[:user_id]
+    @reading_list = Reading.where(user: @user, status: 0).order_by_created_at
```
Comment (NguyenTanDuc) [inline app/controllers/readings_controller.rb]: where -> scope

File: app/controllers/readings_controller.rb [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+    @read_list = Reading.where(user: @user, status: 1).order_by_created_at
```
Comment (NguyenTanDuc) [inline app/controllers/readings_controller.rb]: same

File: app/views/readings/_reading.html.erb [hunk 3]
```diff
@@ -1,12 +1,22 @@
 <% if record = current_user.find_reading(book) %>
 
-  <%= link_to t(:reading_button), user_readings_path(book_id: record.book_id, user_id: current_user.id,
-    behaviour: Settings.book.reading), method: :patch, remote: true,
-    class: record.reading? ? "btn btn-primary btn-reading btn-hidden" : "btn btn-primary btn-reading" %>
-
-  <%= link_to t(:read_button), user_readings_path(book_id: record.book_id, user_id: current_user.id,
-    behaviour: Settings.book.read), method: :patch, remote: true,
-    class: record.read? ? "btn btn-primary btn-read btn-hidden" : "btn btn-primary btn-read" %>
+  <% if record.reading? %>
```
Comment (NguyenTanDuc) [inline app/views/readings/_reading.html.erb]: refactor

File: app/views/readings/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+              <%= read.created_at %>
+            <% end %>
+          </td>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
+
+  <big><big><%= t :what %> <%= @user.email %> <%= t :is_reading %>:</big></big>
+is_reading
```
Comment (NguyenTanDuc) [inline app/views/readings/show.html.erb]: I18n

File: app/views/readings/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+          </td>
+          <td>
+            <%= reading.created_at %>
+          </td>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
+  
+</div>
```
Comment (NguyenTanDuc) [inline app/views/readings/show.html.erb]: new line

File: app/views/readings/show.html.erb [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+      <% @read_list.each do |read| %>
+        <tr class="success">
+          <td>
+            <i><%= read.book.title %></i>
+          </td>
+          <td>
+            <%= read.book.author %>
+          </td>
+          <td>
+            <% if read.updated_at? %>
```
Comment (NguyenTanDuc) [inline app/views/readings/show.html.erb]: refactor

**Comment by NguyenTanDuc** [comment]
OK

File: app/views/readings/show.html.erb [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+  <table class="table table-bordered table-striped table-hover">
+    <thead>
+      <tr>
+        <th class="col-md-5 info header_cell"><%= t :title %></th>
+        <th class="col-md-2 info header_cell"><%= t :author %></th>
+        <th class="col-md-5 info header_cell"><%= t :finish_time %></th>
+      </tr>
+    </thead>
+    <tbody id="read_lists">
+      <% @read_list.each do |read| %>
```
Comment (dieunb) [inline app/views/readings/show.html.erb]: use render partial with collection  to replace 
```
+      <% @read_list.each do |read| %>
+        <%= render "readings/read_item", read: read %>
+      <% end %>

```

File: app/views/readings/show.html.erb [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+  <table class="table table-bordered table-striped table-hover">
+    <thead>
+      <tr>
+        <th class="col-md-5 info header_cell"><%= t :title %></th>
+        <th class="col-md-2 info header_cell"><%= t :author %></th>
+        <th class="col-md-5 info header_cell"><%= t :start_time %></th>
+      </tr>
+    </thead>
+    <tbody id="reading_lists">
+      <% @reading_list.each do |reading| %>
```
Comment (dieunb) [inline app/views/readings/show.html.erb]: As above comment

**Pull request merged**